README.md

```diff
@@ -0,0 +1,56 @@
+# Koop Provider Hub Search OGC
+
+![Coverage](./coverage.svg)
+
+This is a Koop provider that extracts datasets from the ArcGIS Hub Search OGC API.
+
+## Use - Streaming All Datasets
+This provider plugin currently only supports [streaming](https://nodejs.org/api/stream.html#stream_readable_streams) ALL datasets matching a given search query. 
+
+### Define Search Parameters
+To perform a search from an output plugin, attach onto the response.
+```js
+req.res.locals.siteIdentifier = 'my-site.hub.arcgis.com'
+```
+
+
+### Pull the Readable Stream
+Then pass the request object to `this.model.pullStream`.
+```js
+const docStream = await this.model.pullStream(req);
+```
+
+### Full Example
+What you have now is a Node.js `Readable` stream. You can pipe the datasets from the readable through a transform stream in order to format them into some other type of output before sending them back to the browser by piping them to the Express.js response. The following simple example also uses the [`through2`](https://www.npmjs.com/package/through2) library to conveniently define a transform stream.
+
+```js
+async handleRequest (req, res) {
+  req.res.locals.siteIdentifier = 'my-site.hub.arcgis.com'
```

Comment:
So site items also have an `id` field on them (which we are not referencing). To avoid confusion, would it make more sense to change `siteIdentifier` to `siteHostname`?

Reply:
Apologies for the confusion, we have been using `siteIdentifier` as  hub site url for all the output plugins especially for feeds like (dcat, rss) and using in hub feeds api [here](https://devtopia.esri.com/dc/hub-feeds-api/blob/main/src/middlewares/get-feed-template.js#L25). `id` in the request parameter is used as collection key but unforutnately, I don't think koop allows to change the name of request parameter.